elastic-search-support/src/main/resources/application.yml

```diff
@@ -57,6 +58,7 @@ elasticsearch:
     DynamicList: ${elasticsearch.elasticMappings.defaultText}
   casesIndexNameFormat: ${ELASTIC_SEARCH_CASE_INDEX_NAME_FORMAT:%s_cases}
   casesIndexType: _doc
+  # Any changes to casePredefinedMappings need to be reflected in the data-store elasticsearch.case-predefined-mappings.* properties
```

Comment:
# casePredefinedMappings configured as 'defaultText' need to be declared in the data-store elasticsearch.case-predefined-mappings.defaultTextt property